client/src/pages/Login.js

```diff
@@ -0,0 +1,25 @@
+import "./Login.css";
+import Card from "../components/Card";
+import BackButton from "../components/BackButton";
+import FooterSubmit from "../components/Card_components/FooterSubmit";
+import InputField from "../components/InputField";
+import { Link } from "react-router-dom";
+
+export default function Login() {
+  return (
+    <main id="Login" className="card-screen">
+      <Card header={<BackButton />} footer={<FooterSubmit text="Login" />}>
+        <h2 className="margin-b--s">Login</h2>
```

Comment:
this classname sound a bit like chinese to me.. maybe something little more descriptive?